assets/src/dashboard/karma/fixture.js

```diff
@@ -17,7 +17,7 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import * as React from 'react';
```

Comment:
What does this do?

Reply:
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_an_entire_modules_contents

Whereas before it used the default export (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#importing_defaults)

As per the [documentation](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html):

> The default imports will keep working in React 17, but in the longer term we encourage moving away from them.

So we are moving away from them.